scenarios/openshift/container_etcd.yml

```diff
@@ -3,6 +3,6 @@ scenarios:
   namespace: "openshift-etcd"
   label_selector: "k8s-app=etcd"
   container_name: "etcd"
-  action: "kill 1"
+  action: 1
```

Comment:
We need to update https://github.com/redhat-chaos/krkn/blob/main/docs/container_scenarios.md#example-config and https://github.com/redhat-chaos/krkn-hub/blob/main/container-scenarios/env.sh#L9 post this PR merge.